[RHODA][CLI]Adding two CLI tests of deploying instances for Crunchy a…

ikanias opened the pull request:
…nd Cockroach

This PR includes the addition of 2 tests:
1. Deploy CockroachDB Instance using OC CLI
2. Deploy CrunchyDB Instance using OC CLI

This resolves the Jira tickets: DBAAS-688, DBAAS-689

ikanias committed:
[RHODA][CLI]Adding two CLI tests of deploying instances for Crunchy and Cockroach

This PR includes the addition of 2 tests:
1. Deploy CockroachDB Instance using OC CLI
2. Deploy CrunchyDB Instance using OC CLI
3. Added a change by adding the ${isv_name} parameter to have the keyword of deploying an instance more generic

This resolves the Jira tickets: DBAAS-688, DBAAS-689

ikanias committed:
[RHODA][CLI]Adding two CLI tests of deploying instances for Crunchy and Cockroach

This PR includes the addition of 2 tests:
1. Deploy CockroachDB Instance using OC CLI
2. Deploy CrunchyDB Instance using OC CLI

This resolves the Jira tickets: DBAAS-688, DBAAS-689

harshkumarRH commented on resources/keywords/deploy_instance_dev.resource:
```diff
@@ -156,5 +156,11 @@ Text Message For A provider Account With No DB Instances Appears
 User Deploys MongoDB Instance Using CLI
     Deploy Db Instance Cli    Mongo    ${TEST TAGS}
 
+User Deploys CockroachDB Instance using CLI
+    Deploy Db Instance Cli    Cockroach    ${TEST TAGS}
+
+User Deploys CrunchyDB Instance using CLI
+    Deploy Db Instance Cli    crunchy    ${TEST TAGS}
+
```
@ikanias can we try to parameterize the Keyword(User Deploys MongoDB Instance Using CLI) at Line 156 so that we won't need the code from 159 to 164.
IMHO, this would be more efficient
We can have something like this -
`User Deploys ${isv_name} Instance Using CLI
        Deploy Db Instance Cli    ${isv_name}    ${TEST TAGS}`

LMKYT

ikanias replied:
The idea is good, but the problem with this suggestion is that we are adding these keywords to the suits **according to isv type**, and what you are proposing will make us change the structure of the files and re-engineer it because they will not be relevant anymore(tests/cockroachdb_import_connect_cli.robot, tests/crunchydb_import_connect_cli.robot).
WDYT?

mrrajan reviewed (approved):
LGTM

ikanias committed:
Merge branch 'import_provider_account_developer_crdb' of github.com:ikanias/rhoda-ci into import_provider_account_developer_crdb

ikanias committed:
[RHODA][CLI]Adding two tests for deploying instances in developer mode via CLI

This PR contains addition of 2 tests for deploying an instance in developer view, by using CLI.
The tests added are for CrunchyDB and CockroachDB.

Resolves Jira tickets:DBAAS-688, DBAAS-689

ikanias committed:
Merge remote-tracking branch 'origin/import_provider_account_developer_crdb' into import_provider_account_developer_crdb

harshkumarRH reviewed (approved):
Looks good to me
